packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -27,23 +27,23 @@ import { IconRegistry, IIconRegistry } from '@jupyterlab/ui-components';
 
 import { toArray } from '@phosphor/algorithm';
 import { IDragEvent } from '@phosphor/dragdrop';
-import { Widget, PanelLayout } from '@phosphor/widgets';
 
+import {
+  FrontendServices,
+  NotebookParser,
+  SubmissionHandler
+} from '@elyra/application';
 import {
   CommonCanvas,
   CanvasController,
   CommonProperties
 } from '@elyra/canvas';
 import '@elyra/canvas/dist/common-canvas.min.css';
-import {
-  FrontendServices,
-  NotebookParser,
-  SubmissionHandler
-} from '@elyra/application';
+
 import 'carbon-components/css/carbon-components.min.css';
-import '../style/index.css';
 
 import * as palette from './palette.json';
+import { PipelineSubmissionDialog } from './PipelineSubmissionDialog';
```

Comment:
I believe we should have this in a separate "block" above these since this imports a class not `*` @lresende is that consistent with other import orders as you've seen?